students/tonylee/SecondAssignment.py

```diff
@@ -0,0 +1,69 @@
+def exchange_first_last(seq):
+    	s = list(seq)
+	first = seq[0]
+	last = seq[-1]
+	s[0] = last
+	s[-1] = first
```

Comment:
this is using indexing - please use slicing per assignment instructions.

slicing syntax is: [start:end:step]